quickstart/exampledata/config/http_pipeline.yml

```
@@ -1,17 +1,23 @@
 version: 1
+process_count: 2
 
 metrics:
-  enabled: true
+  enabled: false
   port: 8000
 
 input:
   httpinput:
       type: http_input
+      message_backlog_size: 1500000
+      collect_meta: True
+      metafield_name: "@metadata"
       uvicorn_config:
           host: 0.0.0.0
           port: 9000
       endpoints:
-          /lab/123/ABC/auditlog: jsonl
+          /json: json
+          /lab/123/(first|second|third)/js.*: jsonl
+          /lab/123/(ABC|DEF)/pl.*: plaintext
 output:
```

```suggestion
          /lab/123/ABC/auditlog: jsonl
```

I had to add this endpoint to run the implemented quickstart config. Test it with:

```
logprep generate http --input-dir ./quickstart/exampledata/input_logdata/ --target-url http://localhost:9000 --user "" --password "" --events 1000000 --batch-size 1000 --thread_count 5
```